systest/multi-tenancy/basic_test.go

```diff
@@ -351,12 +354,33 @@ func TestLiveLoadMulti(t *testing.T) {
 `,
 		creds: &testutil.LoginParams{UserID: "groot", Passwd: "password", Namespace: ns},
 	}))
-	resp = testutil.QueryData(t, dc1, query3)
-	testutil.CompareJSON(t,
-		`{"me": [{"name":"ns alice"}, {"name": "ns bob"},{"name":"ns chew"},
+
+	err = x.RetryUntilSuccess(10, time.Second, func() error {
+		resp = testutil.QueryData(t, dc1, query3)
+		var o1 interface{}
+		var o2 interface{}
+
+		var err error
+		expected := `{"me": [{"name":"ns alice"}, {"name": "ns bob"},{"name":"ns chew"},
 		{"name": "ns dan"},{"name":"ns eon"}, {"name": "ns free"},{"name":"ns gary"},
-		{"name": "ns hola"}]}`, string(resp))
+		{"name": "ns hola"}]}`
+		err = json.Unmarshal([]byte(expected), &o1)
+		if err != nil {
+			return fmt.Errorf("error unmarshalling string 1 :: %s", err.Error())
+		}
+		err = json.Unmarshal([]byte(string(resp)), &o2)
+		if err != nil {
+			return fmt.Errorf("error unmarshalling string 2 :: %s", err.Error())
+		}
 
+		if reflect.DeepEqual(o1, o2) {
```

Comment:
why do you have to do this instead of CompareJson?

Reply:
i actaully misinterpret this failure. I have removed this code part